crates/rnote-ui/src/workspacebrowser/filerow/actions/mod.rs

```diff
@@ -1,11 +1,13 @@
 // Modules
 mod duplicate;
 mod open;
+mod open_in_default;
```

Comment:
It would be more clear if the module, function and action names are renamed to `open_in_default_app`. It's not much longer anyway.